### PR TITLE
MS Graph Azure AD Reporting app docs

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/integrations/saas-cloud/ms-graph-azure-ad-reporting.md [hunk 1]
```diff
@@ -0,0 +1,333 @@
...
+| count_distinct(id) by duration_in_ms
+| avg(duration_in_ms)
+```
+</details>
+
+## Setup Collection
+
+Follow the instructions for setting up [Microsoft Graph Azure AD Protection](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/ms-graph-azure-ad-reporting-source/) source and use the same source category while installing the app.
+
+## Installing the Microsoft Graph Azure AD Protection app​
```
Comment (JainM6) [inline docs/integrations/saas-cloud/ms-graph-azure-ad-reporting.md:294]:
```suggestion
## Installing the Microsoft Graph Azure AD protection app​
```

File: docs/integrations/saas-cloud/ms-graph-azure-ad-reporting.md [hunk 2]
```diff
@@ -0,0 +1,333 @@
...
+| where initiated_by matches "{{initiated_by}}"
+| where provisioning_action matches "{{provisioning_action}}"
+| where provisioning_status matches "{{provisioning_status}}"
+| where service_principal matches "{{service_principal}}"
+| count_distinct(id) by duration_in_ms
+| avg(duration_in_ms)
+```
+</details>
+
+## Setup Collection
```
Comment (JainM6) [inline docs/integrations/saas-cloud/ms-graph-azure-ad-reporting.md:290]:
```suggestion
## Set up collection
```

File: docs/integrations/saas-cloud/ms-graph-azure-ad-reporting.md [hunk 3]
```diff
@@ -0,0 +1,333 @@
...
+
+Follow the instructions for setting up [Microsoft Graph Azure AD Protection](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/ms-graph-azure-ad-reporting-source/) source and use the same source category while installing the app.
+
+## Installing the Microsoft Graph Azure AD Protection app​
+
+This section has instructions for installing the Microsoft Graph Azure AD Protection app for Sumo Logic and descriptions of each of the dashboards.
```
Comment (JainM6) [inline docs/integrations/saas-cloud/ms-graph-azure-ad-reporting.md:296]:
```suggestion
This section has instructions for installing the Microsoft Graph Azure AD protection app for Sumo Logic and descriptions of each of the dashboards.
```

File: docs/integrations/saas-cloud/ms-graph-azure-ad-reporting.md [hunk 4]
```diff
@@ -0,0 +1,333 @@
...
+1. Complete the following fields:
+   1. **Data Source**. Select either of these options for the data source:
+      * Choose **Source Category** and then choose a source category from the list.
+      * Select **Enter a Custom Data Filter** and type in a custom source category that starts with an underscore. For Example, `_sourceCategory=MyCategory`.
+    2. **Folder Name**. You can retain the existing name, or enter a name of your choice for the app.
+    3. Select the **Location in Library** (the default is the **Personal** folder in the library), or click **New Folder** to add a new folder.
+1. Click **Next**.
+
+Once an app is installed, it will appear in your **Personal** folder, or other folder that you specified. You can share it with your organization.
+
```
Comment (JainM6) [inline docs/integrations/saas-cloud/ms-graph-azure-ad-reporting.md:311]:
```suggestion
```

File: docs/integrations/saas-cloud/ms-graph-azure-ad-reporting.md [hunk 5]
```diff
@@ -0,0 +1,333 @@
...
+The panels will begin to fill automatically. It's worth noting that each panel gradually fills with data that matches the time range query and has been received since the panel was created. The results will not be available right away, but with some patience, you will be able to view full graphs and maps.
+
+## Viewing Microsoft Graph Azure AD Protection dashboards​​
```
[the remark this answers is not in the document]
Comment (JainM6) [reply docs/integrations/saas-cloud/ms-graph-azure-ad-reporting.md:314]: "protection" should be in small caps"

**Review by JainM6 — CHANGES_REQUESTED** [review]
Minor formatting changes..

File: docs/integrations/saas-cloud/ms-graph-azure-ad-reporting.md (hunk 1 above)
Comment (JV0812) [reply docs/integrations/saas-cloud/ms-graph-azure-ad-reporting.md]: Changed as "Reporting". And the letter R is in caps as it is a part of app name.

File: docs/integrations/saas-cloud/ms-graph-azure-ad-reporting.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (JV0812) [reply docs/integrations/saas-cloud/ms-graph-azure-ad-reporting.md]: Changed as "Reporting". And the letter R is in caps as it is a part of app name.

File: docs/integrations/saas-cloud/ms-graph-azure-ad-reporting.md (hunk 3 above)
Comment (JV0812) [reply docs/integrations/saas-cloud/ms-graph-azure-ad-reporting.md]: Changed as "Reporting". And the letter R is in caps as it is a part of app name.

File: docs/integrations/saas-cloud/ms-graph-azure-ad-reporting.md [hunk 6]
```diff
@@ -0,0 +1,332 @@
...
+- **Identify Suspicious Activities**. Identify suspicious activity, such as sign-ins from unfamiliar/risky locations or multiple sign-in attempts.
+- **Analyze Provisioning Activities**. Provides distribution of provisioning activities by status, actions, and initiators. It also shows the most frequent service principals used while provisioning.
+
+## Log Types
+
+The App uses Microsoft Graph Azure AD Reporting Source to collect [Audit](https://learn.microsoft.com/en-us/graph/api/resources/directoryaudit?view=graph-rest-1.0), [Sign-in](https://learn.microsoft.com/en-us/graph/api/resources/signin?view=graph-rest-1.0), and [Provisioning](https://learn.microsoft.com/en-us/graph/api/resources/provisioningobjectsummary?view=graph-rest-1.0) activities.
+
+## Sample Log Message
+
+<details><summary>Click here to expand</summary>
```
Comment (JainM6) [inline docs/integrations/saas-cloud/ms-graph-azure-ad-reporting.md:25]:
```suggestion
<details><summary>View Sample Log Message</summary>
```

Comment (JainM6) [reply docs/integrations/saas-cloud/ms-graph-azure-ad-reporting.md:25]: I think it's better if we change the expand button text from"Click here to expand" to View Sample Log Message.

File: docs/integrations/saas-cloud/ms-graph-azure-ad-reporting.md [hunk 7]
```diff
@@ -0,0 +1,332 @@
...
+    "displayName": "",
+    "details": {}
+  }
+}
+```
+</details>
+
+## Sample Query
+
+<details><summary>Click here to expand</summary>
```
Comment (JainM6) [inline docs/integrations/saas-cloud/ms-graph-azure-ad-reporting.md:248]:
```suggestion
<details><summary>View Sample Queries</summary>
```

**Review by JainM6 — APPROVED** [review]
LTGM

**Pull request merged**